src/workerd/server/bundle-fs.c++

```diff
@@ -0,0 +1,80 @@
+#include "bundle-fs.h"
+
+#include <capnp/message.h>
+
+namespace workerd::server {
+kj::Rc<Directory> getBundleDirectory(config::Worker::Reader conf) {
+  static kj::Path kRoot{};
```

Comment:
Not sure why this should be static or why this needs to be outside of the lambda, if this is just an optimization then I'd vote to just remove the opt and and move it into the lambda.
If you insist then let's at least make it const.